web/client/components/map/openlayers/__tests__/Map-test.jsx

```diff
@@ -891,4 +891,120 @@ describe('OpenlayersMap', () => {
         attributions = document.getElementsByClassName('ol-attribution');
         expect(attributions.length).toBe(2);
     });
+
+    it('test updateMapInfoState with custom projection 3003 (not listed in wrappedProjections)', () => {
+
+        const CENTER_OUTSIDE_OF_MAX_EXTENT = {
+            x: 50,
+            y: 70,
+            crs: 'EPSG:4326'
+        };
+        const testHandlers = {
```

Comment:
I would change the handler name and related spy to a more specific one, maybe:
spy  ==> spyOnMapViewChanges
handler ==> onMapViewChanges

if you prefer to use a more general names is fine too.